File: Programming EF Code First/chapter4/Chapter4/BAGA_FluentAPI/DataAccess/PersonConfiguration.cs
using System.Data.Entity.ModelConfiguration;
using Model;
using System.ComponentModel.DataAnnotations;

namespace DataAccess
{
  public class PersonConfiguration : EntityTypeConfiguration<Person>
  {
    public PersonConfiguration()
    {
      Property(p => p.SocialSecurityNumber)
        .IsConcurrencyToken();
    }
  }
}

File: Programming EF Code First/chapter3/Chapter 3 VB/Fluent/DataAccess/PersonConfiguration.vb
Imports System.Data.Entity.ModelConfiguration
Imports Model
Imports System.ComponentModel.DataAnnotations

Namespace DataAccess
  Public Class PersonConfiguration
	  Inherits EntityTypeConfiguration(Of Person)
	Public Sub New()
      Me.Property(Function(p) p.SocialSecurityNumber).IsConcurrencyToken()
	End Sub
  End Class
End Namespace


